src/main/groovy/org/shipkit/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -52,6 +52,18 @@ public void apply(final Project project) {
 
         project.allprojects(new Action<Project>() {
             @Override
+            public void execute(final Project project) {
+                project.getPlugins().withId("java", new Action<Plugin>() {
```

Comment:
I suggest we move it to the existing 'allprojects' block. This way we have less code to maintain and the execution is a bit faster (no need to iterate projects multiple times).